src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -735,7 +735,7 @@ private IntPtr WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wPar
                     break;
 
                 case WindowMessage.ERASEBKGND:
-                    return new IntPtr(1);
+                    break;
```

Comment:
Ok if that's true can you add a comment here (in the code) explaining that behaviour.